src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -30,37 +36,77 @@ User attributes are not required by default. You can make user attributes requir
 
 </Callout>
 
-#### How will you verify users at sign-up?
+### How will you verify users at sign-up?
 
 Once the initial sign-up is done, the user will need to verify and confirm their contact information before their account is activated. This is done by sending a verification code via email or phone number that can be collected on initial sign-up. Email verification asks the user to click a link or enter a code sent to their email address provided during sign-up. Phone verification is done by asking the user to enter the code sent to them in a SMS text message. You can also enable auto-confirmation to skip this step and automatically confirm the user after sign-up. This is accomplished using the `PreSignUp` Lambda trigger which is explained in the [Amazon Cognito documentation](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-pre-sign-up.html).
  
-#### Do you need local or global sign-out?
+### Do you need local or global sign-out?
```

Comment:
looking at this whole section, I feel like it would make sense to just explain it at the logout section - and what we have on there is enough. This part here is overkill.

Reply:
Yep - taking the global signout part here out.